www/source/partials/docs/_glossary-keys.html.md.erb

```diff
@@ -2,13 +2,20 @@
 
 Habitat has strong cryptography built into both Habitat Builder and the Supervisor. This means there are several different kinds of keys.
 
-## Origin Keys
+## Origin Key Pairs
 
-As described in the [packages documentation](/docs/glossary/#glossary-packages), every package in Habitat belongs to an origin and is cryptographically signed with that origin's private key.
+Every package and artifact in Habitat belongs to an [origin](/docs/glossary/#glossary-packages) and is cryptographically signed with that origin's private key.
+Habitat requires both the public and the private key for producing and consuming with origin packages.
+On the production side, Builder---both the service and the Depot---use the origin private key to sign and verify packages as they move from your development environment to Builder. On the consumption side, the Supervisors require the public key to verify the compiled artifacts.
 
 Origin key cryptography is asymmetric: it has a public key that you can distribute freely, and a private key that you should keep safe.
 
-Supervisors, by default, will refuse to run packages for which they do not have the public key. They use this public key to verify the integrity of the Habitat package they download before running it. Supervisors can be provided the public key by pointing them at a depot that has it, or by putting the key on disk outside of Habitat.
+Habitat uses the public origin key to verify the integrity of downloaded artifacts before running them.
+Habitat will only run artifacts for which it has the public origin key, and will not run them without it.
```

Comment:
Verification of a `.hart` occurs when it is installed and has nothing to do with running them. Every `.hart` is subject to verification, but not every `.hart` can be "run" in the same way. If it's a `.hart` for a package which defines a service, it can be run with `hab svc load`, but some packages contain non-service binaries, potentially many of them, which can be run individually via `hab pkg exec <CMD>`. Still other packages define libraries and have no concept of being "run" at all.

Reply:
changed to "install"